blocks/file-blocks/edit/index.tsx

```diff
@@ -12,6 +12,8 @@ import { Button, FormControl, TextInput } from "@primer/react";
 
 export default function (props: FileBlockProps) {
   const { content, context, onUpdateContent } = props;
+  const onFetchInternalEndpoint =
+    props.private__onFetchInternalEndpoint || onFetchInternalEndpointPolyfill;
```

Comment:
this is temporary for backward compatibility right?

Reply:
yep!